assignment.md

```diff
@@ -1,9 +1,44 @@
-## Content
+# What is the difference between `git` push, pull, and fetch?
 
-### What is the difference between push, pull, and fetch?
+The below commands are a fundamental part of any developer's Git workflow:
 
-- `git push` - sent changes from a local branch to a remote repo
-- `git fetch` - get changes from a remote repo into your tracking branch
-- `git pull` - will get changes from a remote branch into your tracking branch and merge them into a local branch
+- `git push` - The `git push` command _pushes_ a local branch's content to a remote repository.
+- `git fetch` - The `git fetch` command _fetches_ content from a branch in a remote repository.
+- `git pull` - The `git pull` command fetches content from a branch in a remote repository _and merges_ that content into your local branch.
 
```

Comment:
How would you feel about inserting a diagram here so the readers have a visual representation to reference? Maybe something in the same style as: https://www.janbasktraining.com/blog/uploads/images/image_750x_5d76346f0fbf8.jpg?

If that sounds good, I can help you create a diagram, or I can make one and send it to you for feedback.